workflows/diagnostics/fv3net/diagnostics/prognostic_run/shell.py

```diff
@@ -0,0 +1,197 @@
+import cmd
+from fv3net.diagnostics.prognostic_run import load_run_data
+import intake
+import vcm.catalog
+import vcm
+import xarray as xr
+import fv3viz
+import pathlib
+import matplotlib.pyplot as plt
+import sys
+import io
+import warnings
+
+from . import iterm
+
+warnings.filterwarnings("ignore")
+
+
+def meridional_transect(ds: xr.Dataset, lon):
+    transect_coords = vcm.select.meridional_ring(lon)
+    ds = vcm.interpolate_unstructured(ds, transect_coords)
+    return ds.swap_dims({"sample": "lat"})
+
+
+class PlotTape:
+    def __init__(self):
+        self.count = 0
+
+    def save_plot(self):
+        filename = f"image_{self.count}.png"
+        plt.savefig(filename)
+        plt.close(plt.gcf())
+        self.count += 1
+
+
+class OneFileTape:
+    """Useful for working in vscode...updates file in place"""
+
+    def save_plot(self):
+        filename = f"image.png"
+        plt.savefig(filename)
+        plt.close(plt.gcf())
+
+
+class ItermTape:
+    width = 70
+
+    def save_plot(self):
+        f = io.BytesIO()
+        plt.savefig(f)
+        iterm.write_image(
+            f.getvalue(),
+            sys.stderr.buffer,
+            filename="file",
+            width=self.width,
+            preserve_aspect_ratio=True,
+        )
+        plt.close(plt.gcf())
+
+
+class State:
+    def __init__(self):
+        self.data_3d = None
+        self.data_2d = None
+        self.tape = OneFileTape()
+
+    def load(self, url):
+        prognostic = load_run_data.SegmentedRun(url, catalog)
+        self.data_3d = prognostic.data_3d
+        self.data_2d = prognostic.data_2d.merge(grid.drop("area"))
```

Comment:
Would be better to use area from the catalog grid. It can be corrupted in prog run data if the run crashed.